chrome_manifest_v3/views/class_popupview.js

```
@@ -1480,8 +1478,8 @@ class PopupView {
         'Board/List data may be stale. You can try reloading your Trello boards.';
     }
 
-    if (this.data && this.data.newCard) {
-      resp.title = this.data.newCard.title; // Put a temp copy of this over where we'll get the other data
+    if (this.state && this.state.title) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)